Backend/Backend/Program.cs

```diff
@@ -119,6 +121,9 @@ public static void Main(string[] args)
                 .AllowAnyMethod();
         }));
 
+        builder.Logging.AddConsole();
+        builder.Logging.AddDebug();
```

Comment:
why are you adding the console here? Logging is already setup to log to a file, this aint needed

Reply:
I need to see it when debugging. I think it is easier than opening file each time idk